src/datasources/asset-calibration/AssetCalibrationDataSource.test.ts

```diff
@@ -56,9 +58,9 @@ const locationGroupCalibrationForecastResponseMock: CalibrationForecastResponse
 {
   calibrationForecast: {
     columns: [
-      { name: "", values: [1], columnDescriptors: [{ value: "Location1", type: ColumnDescriptorType.StringValue }] },
-      { name: "", values: [2], columnDescriptors: [{ value: "Location2", type: ColumnDescriptorType.StringValue }] },
-      { name: "", values: [3], columnDescriptors: [{ value: "Location3", type: ColumnDescriptorType.StringValue }] }
+      { name: "", values: [1], columnDescriptors: [{ value: "Location1", type: ColumnDescriptorType.MinionId }] },
```

Comment:
Please create a new test case with MinionId.  Don't overwrite the current test because it's still a valid test. Location can be Both StringValue and MinionId.